### PR TITLE
Version1

### PR DESCRIPTION
**Pull request by Brindha-Josh**
I have updated the coding as per ur changes and created pull request again
1.Removed  CUSTOMERAPISQL.csproj.user: in git
2.Activated the route [Route("")]
3.The namespace should not be in all capitals.. (change in all your code-files)-Updated to CustomerMgmt
4.Customer1Controller is renamed as CustomerController
5.removed  the  Route [Route("")] in  Controllers/Customer1Controller.cs: for Edit/Delete/Post
6.In Controllers/LoginController.cs: removed Post route-name
7.In Models/Customer1.cs:-renamed as Customer
8.In Models/UserModel.cs:-Aligned the coding.

### CODE REVIEW

**Comment by perkare** [comment]
Some comments: 
- You should not remove the .csproj file. Only the .csproj.user file
- Make sure the branch builds and has no failing checks (the pull request nor reports 2 failing checks)

File: Controllers/CustomerController.cs [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using System.Text;
+using Microsoft.IdentityModel.Tokens;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using Microsoft.AspNetCore.Identity;
+namespace CustomerMgmt.Controllers
+{
+
+    [Route("")]
```
Comment (perkare) [inline Controllers/CustomerController.cs]: You can remove this. Here and all other places you may have used it. Use conventional routing.
Use app.UseMvcWithDefaultRoute(); in Startup.cs
See https://docs.microsoft.com/en-us/aspnet/core/mvc/controllers/routing?view=aspnetcore-3.1

**Review by perkare — CHANGES_REQUESTED** [review]
A few more issues to fix:

- Delete /Models/Customers.cs (this is not in use is it?)
- Rename /Models/UserModel.cs to /Models/User.cs
- Align code ("format document") in /Startup.cs
- Remove out commented code in /Startup.cs
- You can probably remove the code in  `endpoints.MapControllers(...)` and  replace it with `app.UseMvcWithDefaultRoute();` in Startup.cs? (please correct me if I am wrong)
- Why do you have two azure-pipelines files? Do you need both?

**Comment by perkare** [comment]
Why are you adding CustomerData.cs as a data access layer when you already are using EntityFramework for this? Can you remove this and use your existing data-context instead?

**Comment by Brindha-Josh** [comment]
Hi
I was just trying to fetch data using that method.will delete that form and
try to access use that entity layer.will upload the form in git hub.

Regards
Brindha

**Review by perkare — APPROVED** [review]

**Pull request merged**